tests/libres_tests/res/config/test_config.py

```diff
@@ -60,15 +60,6 @@ class ConfigTest(ResTest):
     def setUp(self):
         self.file_list = []
 
-    def test_enums(self):
-        source_file_path = "libres/lib/include/ert/config/config_schema_item.hpp"
-        self.assertEnumIsFullyDefined(
-            ContentTypeEnum, "config_item_types", source_file_path
-        )
-        self.assertEnumIsFullyDefined(
```

Comment:
Think I agree with removing these, but is it needed in this PR? Might make sense to split it out and only have things that actually relate to moving code in this PR, and your other changes in another PR.

Reply:
It's either remove or fix the path to libres. I went with the former.